webapp/tests/Unit/Controller/Jury/TeamControllerTest.php

```diff
@@ -28,7 +28,7 @@ class TeamControllerTest extends JuryControllerTest
                                                            'penalty' => '0',
                                                            'room' => 'The first room',
                                                            'internalcomments' => 'This is a team without a user',
-                                                           'contests' => ['1'],
+                                                           'contests' => [],
```

Comment:
Is this because we don't have the demo contest anymore in the unit tests?

Reply:
Yes, see my comment about unit tests not being unit tests here: https://github.com/DOMjudge/domjudge/pull/1690#issuecomment-1295752606

We should fix this for all unit tests IMHO. But independently from this PR.